packages/lib/universal/upload/get-file.ts

```diff
@@ -0,0 +1,51 @@
+import { base64 } from '@scure/base';
+import { match } from 'ts-pattern';
+
+import { DocumentDataType } from '@documenso/prisma/client';
+
+import { getPresignGetUrl } from './server-actions';
+
+export type GetFileOptions = {
+  type: DocumentDataType;
+  data: string;
+};
+
+export const getFile = async ({ type, data }: GetFileOptions) => {
```

Comment:
what happens if a file doesn't match any format?

Reply:
The `.exhaustive()` flag turns it into a build time error 😄

Reply:
TIL, thanks!